### ISSUE TITLE
Not clear which prepublish items are clickable

### ISSUE THREAD

**Issue by embarks** [GoogleForCreators/web-stories-wp#5923]
Per 1.3-b bug bash:
- Add an underline and a title to the clickable items. 
- The pointer cursor should only be on the underlined part (the title of the issue).

**Comment by csossi** [comment]
Verified in QA (checked various violations)
